src/MultiRegion/cubed_sphere_boundary_conditions.jl

```diff
@@ -238,25 +238,27 @@ function fill_cubed_sphere_halo_regions!(field_1::CubedSphereField{<:Face, <:Cen
         end
     end
 
-    #-- Add one valid field_1, field_2 value next to the corner, that allows
-    #   to compute vorticity on a wider stencil (e.g., vort3(0,1) & (1,0)).
-    for region in 1:6
-        for k in -Hz+1:Nz+Hz
-            #- SW corner:
-            field_1[region][1-Hc:0, 0, k] .= field_2[region][1, 1-Hc:0, k]
-            field_2[region][0, 1-Hc:0, k] .= field_1[region][1-Hc:0, 1, k]'
-        end
-        if Hc > 1
+    if signed
```

Comment:
why don't we want this when `signed = false`?

Reply:
A better approach involves introducing an additional keyword argument, `velocities=true`, and then utilizing the condition `if velocities`. But as it turns out, while filling the halos of paired grid coordinates and metrics, we always set `signed = false`. This is because these paired quantities are not the horizontal components of a vector like velocity, and therefore, sign reversal does not apply when these components are swapped. In practice, we only use `signed = true` while filling the halos of velocities, in which case the velocity halos adjacent to the corners are essential for computing vorticity.

Reply:
I’m sorry, I don’t quite get it.
Please add a comment informing our future selves or others looking at this?

Reply:
Done. 🙂

Reply:
I have an issue here: this part is intended to be able to compute vorticity over a wider range, e.g., if someone wants to use biharmonic viscosity. But to compute vorticity, I will also need the corresponding grid-metric dx^fc, dy^cf to be right to multiply the 2 velocity components.

Reply:
@jm-c, I can remove the conditional in that case. Let's discuss it during the meeting tomorrow morning.

Reply:
@siddharthabishnu I appreciate the comment, but I'm not sure I understand it.
show it to @glwagner or @simone-silvestri who haven't been part of the discussion -- do they understand it?